src/tests/unit/tests/injected/__snapshots__/layered-details-dialog-component.test.tsx.snap

```diff
@@ -1,17 +1,16 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`LayeredDetailsDialogComponent renders per snapshot 1`] = `
-<LayerHost
-  dir="ltr"
-  id="insights-dialog-layer-host"
->
-  <DetailsDialog
-    deps={
-      {
-        "getRTL": [Function],
-      }
-    }
-    devToolActionMessageCreator="devToolActionMessageCreator"
-  />
-</LayerHost>
+<DocumentFragment>
+  <div
+    class="ms-LayerHost"
+    dir="ltr"
+    id="insights-dialog-layer-host"
+  >
+    <mock-detailsdialog
```

Comment:
can we use expectMockedComponentPropsToMatchSnapshot(DetailsDialog)?

Reply:
Hi @JeevaniChinthala  there are some build errors please check it.
Also i request you to please have a selft review of PR and you can identify where we can use expectMockedComponentPropsToMatchSnapshot etc utilities so that we can make snapshot matching with old, please check. I can see we can make some improvements for snapshots

Reply:
> can we use expectMockedComponentPropsToMatchSnapshot(DetailsDialog)?

Yes, implemented.

Reply:
> Hi @JeevaniChinthala there are some build errors please check it. Also i request you to please have a selft review of PR and you can identify where we can use expectMockedComponentPropsToMatchSnapshot etc utilities so that we can make snapshot matching with old, please check. I can see we can make some improvements for snapshots

Sure